From f79f14a2b777f20150d4ffa9bb0ec1cf77f786f0 Mon Sep 17 00:00:00 2001
From: Samuel Holland <samuel@sholland.org>
Date: Mon, 31 Oct 2022 22:14:36 -0500
Subject: [PATCH 19/68] sunxi: Hide the SUNXI_MINIMUM_DRAM_MB symbol

This option affects the ABI between SPL/U-Boot and U-Boot/scripts, so it
should not normally be changed by the user.

Signed-off-by: Samuel Holland <samuel@sholland.org>
---
 board/sunxi/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/board/sunxi/Kconfig
+++ b/board/sunxi/Kconfig
@@ -69,7 +69,7 @@ config SPL_STACK_R_ADDR
 	default 0x4fe00000 if SUNXI_GEN_NCAT2
 
 config SUNXI_MINIMUM_DRAM_MB
-	int "minimum DRAM size"
+	int
 	default 32 if MACH_SUNIV
 	default 64 if MACH_SUN8I_V3S
 	default 256
